diff --git a/node_modules/@docusaurus/core/LICENSE b/node_modules/@docusaurus/core/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/README.md b/node_modules/@docusaurus/core/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/bin/beforeCli.mjs b/node_modules/@docusaurus/core/bin/beforeCli.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/babel/preset.d.ts b/node_modules/@docusaurus/core/lib/babel/preset.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/babel/preset.js b/node_modules/@docusaurus/core/lib/babel/preset.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/App.d.ts b/node_modules/@docusaurus/core/lib/client/App.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/App.js b/node_modules/@docusaurus/core/lib/client/App.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/BaseUrlIssueBanner/index.d.ts b/node_modules/@docusaurus/core/lib/client/BaseUrlIssueBanner/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/BaseUrlIssueBanner/index.js b/node_modules/@docusaurus/core/lib/client/BaseUrlIssueBanner/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/BaseUrlIssueBanner/styles.module.css b/node_modules/@docusaurus/core/lib/client/BaseUrlIssueBanner/styles.module.css
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/BrokenLinksContext.d.ts b/node_modules/@docusaurus/core/lib/client/BrokenLinksContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/BrokenLinksContext.js b/node_modules/@docusaurus/core/lib/client/BrokenLinksContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/ClientLifecyclesDispatcher.d.ts b/node_modules/@docusaurus/core/lib/client/ClientLifecyclesDispatcher.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/ClientLifecyclesDispatcher.js b/node_modules/@docusaurus/core/lib/client/ClientLifecyclesDispatcher.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/PendingNavigation.d.ts b/node_modules/@docusaurus/core/lib/client/PendingNavigation.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/PendingNavigation.js b/node_modules/@docusaurus/core/lib/client/PendingNavigation.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/SiteMetadataDefaults.d.ts b/node_modules/@docusaurus/core/lib/client/SiteMetadataDefaults.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/SiteMetadataDefaults.js b/node_modules/@docusaurus/core/lib/client/SiteMetadataDefaults.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/browserContext.d.ts b/node_modules/@docusaurus/core/lib/client/browserContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/browserContext.js b/node_modules/@docusaurus/core/lib/client/browserContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/clientEntry.d.ts b/node_modules/@docusaurus/core/lib/client/clientEntry.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/clientEntry.js b/node_modules/@docusaurus/core/lib/client/clientEntry.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/docusaurus.d.ts b/node_modules/@docusaurus/core/lib/client/docusaurus.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/docusaurus.js b/node_modules/@docusaurus/core/lib/client/docusaurus.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/docusaurusContext.d.ts b/node_modules/@docusaurus/core/lib/client/docusaurusContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/docusaurusContext.js b/node_modules/@docusaurus/core/lib/client/docusaurusContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/BrowserOnly.d.ts b/node_modules/@docusaurus/core/lib/client/exports/BrowserOnly.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/BrowserOnly.js b/node_modules/@docusaurus/core/lib/client/exports/BrowserOnly.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/ComponentCreator.d.ts b/node_modules/@docusaurus/core/lib/client/exports/ComponentCreator.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/ComponentCreator.js b/node_modules/@docusaurus/core/lib/client/exports/ComponentCreator.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/ErrorBoundary.d.ts b/node_modules/@docusaurus/core/lib/client/exports/ErrorBoundary.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/ErrorBoundary.js b/node_modules/@docusaurus/core/lib/client/exports/ErrorBoundary.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/ExecutionEnvironment.d.ts b/node_modules/@docusaurus/core/lib/client/exports/ExecutionEnvironment.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/ExecutionEnvironment.js b/node_modules/@docusaurus/core/lib/client/exports/ExecutionEnvironment.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Head.d.ts b/node_modules/@docusaurus/core/lib/client/exports/Head.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Head.js b/node_modules/@docusaurus/core/lib/client/exports/Head.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Interpolate.d.ts b/node_modules/@docusaurus/core/lib/client/exports/Interpolate.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Interpolate.js b/node_modules/@docusaurus/core/lib/client/exports/Interpolate.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Link.d.ts b/node_modules/@docusaurus/core/lib/client/exports/Link.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Link.js b/node_modules/@docusaurus/core/lib/client/exports/Link.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Noop.d.ts b/node_modules/@docusaurus/core/lib/client/exports/Noop.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Noop.js b/node_modules/@docusaurus/core/lib/client/exports/Noop.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Translate.d.ts b/node_modules/@docusaurus/core/lib/client/exports/Translate.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/Translate.js b/node_modules/@docusaurus/core/lib/client/exports/Translate.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/constants.d.ts b/node_modules/@docusaurus/core/lib/client/exports/constants.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/constants.js b/node_modules/@docusaurus/core/lib/client/exports/constants.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/isInternalUrl.d.ts b/node_modules/@docusaurus/core/lib/client/exports/isInternalUrl.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/isInternalUrl.js b/node_modules/@docusaurus/core/lib/client/exports/isInternalUrl.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/renderRoutes.d.ts b/node_modules/@docusaurus/core/lib/client/exports/renderRoutes.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/renderRoutes.js b/node_modules/@docusaurus/core/lib/client/exports/renderRoutes.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/router.d.ts b/node_modules/@docusaurus/core/lib/client/exports/router.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/router.js b/node_modules/@docusaurus/core/lib/client/exports/router.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useBaseUrl.d.ts b/node_modules/@docusaurus/core/lib/client/exports/useBaseUrl.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useBaseUrl.js b/node_modules/@docusaurus/core/lib/client/exports/useBaseUrl.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useBrokenLinks.d.ts b/node_modules/@docusaurus/core/lib/client/exports/useBrokenLinks.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useBrokenLinks.js b/node_modules/@docusaurus/core/lib/client/exports/useBrokenLinks.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useDocusaurusContext.d.ts b/node_modules/@docusaurus/core/lib/client/exports/useDocusaurusContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useDocusaurusContext.js b/node_modules/@docusaurus/core/lib/client/exports/useDocusaurusContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useGlobalData.d.ts b/node_modules/@docusaurus/core/lib/client/exports/useGlobalData.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useGlobalData.js b/node_modules/@docusaurus/core/lib/client/exports/useGlobalData.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useIsBrowser.d.ts b/node_modules/@docusaurus/core/lib/client/exports/useIsBrowser.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useIsBrowser.js b/node_modules/@docusaurus/core/lib/client/exports/useIsBrowser.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useIsomorphicLayoutEffect.d.ts b/node_modules/@docusaurus/core/lib/client/exports/useIsomorphicLayoutEffect.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useIsomorphicLayoutEffect.js b/node_modules/@docusaurus/core/lib/client/exports/useIsomorphicLayoutEffect.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useRouteContext.d.ts b/node_modules/@docusaurus/core/lib/client/exports/useRouteContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/exports/useRouteContext.js b/node_modules/@docusaurus/core/lib/client/exports/useRouteContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/flat.d.ts b/node_modules/@docusaurus/core/lib/client/flat.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/flat.js b/node_modules/@docusaurus/core/lib/client/flat.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/hasHydratedDataAttribute.d.ts b/node_modules/@docusaurus/core/lib/client/hasHydratedDataAttribute.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/hasHydratedDataAttribute.js b/node_modules/@docusaurus/core/lib/client/hasHydratedDataAttribute.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/normalizeLocation.d.ts b/node_modules/@docusaurus/core/lib/client/normalizeLocation.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/normalizeLocation.js b/node_modules/@docusaurus/core/lib/client/normalizeLocation.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/prefetch.d.ts b/node_modules/@docusaurus/core/lib/client/prefetch.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/prefetch.js b/node_modules/@docusaurus/core/lib/client/prefetch.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/preload.d.ts b/node_modules/@docusaurus/core/lib/client/preload.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/preload.js b/node_modules/@docusaurus/core/lib/client/preload.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/renderToHtml.d.ts b/node_modules/@docusaurus/core/lib/client/renderToHtml.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/renderToHtml.js b/node_modules/@docusaurus/core/lib/client/renderToHtml.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/routeContext.d.ts b/node_modules/@docusaurus/core/lib/client/routeContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/routeContext.js b/node_modules/@docusaurus/core/lib/client/routeContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/serverEntry.d.ts b/node_modules/@docusaurus/core/lib/client/serverEntry.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/serverEntry.js b/node_modules/@docusaurus/core/lib/client/serverEntry.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Error/index.d.ts b/node_modules/@docusaurus/core/lib/client/theme-fallback/Error/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Error/index.js b/node_modules/@docusaurus/core/lib/client/theme-fallback/Error/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Layout/index.d.ts b/node_modules/@docusaurus/core/lib/client/theme-fallback/Layout/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Layout/index.js b/node_modules/@docusaurus/core/lib/client/theme-fallback/Layout/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Loading/index.d.ts b/node_modules/@docusaurus/core/lib/client/theme-fallback/Loading/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Loading/index.js b/node_modules/@docusaurus/core/lib/client/theme-fallback/Loading/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/NotFound/index.d.ts b/node_modules/@docusaurus/core/lib/client/theme-fallback/NotFound/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/NotFound/index.js b/node_modules/@docusaurus/core/lib/client/theme-fallback/NotFound/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Root/index.d.ts b/node_modules/@docusaurus/core/lib/client/theme-fallback/Root/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/Root/index.js b/node_modules/@docusaurus/core/lib/client/theme-fallback/Root/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/SiteMetadata/index.d.ts b/node_modules/@docusaurus/core/lib/client/theme-fallback/SiteMetadata/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/client/theme-fallback/SiteMetadata/index.js b/node_modules/@docusaurus/core/lib/client/theme-fallback/SiteMetadata/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/build/build.d.ts b/node_modules/@docusaurus/core/lib/commands/build/build.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/build/build.js b/node_modules/@docusaurus/core/lib/commands/build/build.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/build/buildLocale.d.ts b/node_modules/@docusaurus/core/lib/commands/build/buildLocale.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/build/buildLocale.js b/node_modules/@docusaurus/core/lib/commands/build/buildLocale.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/clear.d.ts b/node_modules/@docusaurus/core/lib/commands/clear.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/clear.js b/node_modules/@docusaurus/core/lib/commands/clear.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/cli.d.ts b/node_modules/@docusaurus/core/lib/commands/cli.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/cli.js b/node_modules/@docusaurus/core/lib/commands/cli.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/deploy.d.ts b/node_modules/@docusaurus/core/lib/commands/deploy.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/deploy.js b/node_modules/@docusaurus/core/lib/commands/deploy.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/external.d.ts b/node_modules/@docusaurus/core/lib/commands/external.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/external.js b/node_modules/@docusaurus/core/lib/commands/external.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/serve.d.ts b/node_modules/@docusaurus/core/lib/commands/serve.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/serve.js b/node_modules/@docusaurus/core/lib/commands/serve.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/start.d.ts b/node_modules/@docusaurus/core/lib/commands/start/start.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/start.js b/node_modules/@docusaurus/core/lib/commands/start/start.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/utils.d.ts b/node_modules/@docusaurus/core/lib/commands/start/utils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/utils.js b/node_modules/@docusaurus/core/lib/commands/start/utils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/watcher.d.ts b/node_modules/@docusaurus/core/lib/commands/start/watcher.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/watcher.js b/node_modules/@docusaurus/core/lib/commands/start/watcher.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/webpack.d.ts b/node_modules/@docusaurus/core/lib/commands/start/webpack.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/start/webpack.js b/node_modules/@docusaurus/core/lib/commands/start/webpack.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/actions.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/actions.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/actions.js b/node_modules/@docusaurus/core/lib/commands/swizzle/actions.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/common.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/common.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/common.js b/node_modules/@docusaurus/core/lib/commands/swizzle/common.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/components.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/components.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/components.js b/node_modules/@docusaurus/core/lib/commands/swizzle/components.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/config.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/config.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/config.js b/node_modules/@docusaurus/core/lib/commands/swizzle/config.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/context.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/context.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/context.js b/node_modules/@docusaurus/core/lib/commands/swizzle/context.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/index.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/index.js b/node_modules/@docusaurus/core/lib/commands/swizzle/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/prompts.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/prompts.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/prompts.js b/node_modules/@docusaurus/core/lib/commands/swizzle/prompts.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/tables.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/tables.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/tables.js b/node_modules/@docusaurus/core/lib/commands/swizzle/tables.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/themes.d.ts b/node_modules/@docusaurus/core/lib/commands/swizzle/themes.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/swizzle/themes.js b/node_modules/@docusaurus/core/lib/commands/swizzle/themes.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/writeHeadingIds.d.ts b/node_modules/@docusaurus/core/lib/commands/writeHeadingIds.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/writeHeadingIds.js b/node_modules/@docusaurus/core/lib/commands/writeHeadingIds.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/writeTranslations.d.ts b/node_modules/@docusaurus/core/lib/commands/writeTranslations.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/commands/writeTranslations.js b/node_modules/@docusaurus/core/lib/commands/writeTranslations.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/index.d.ts b/node_modules/@docusaurus/core/lib/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/index.js b/node_modules/@docusaurus/core/lib/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/brokenLinks.d.ts b/node_modules/@docusaurus/core/lib/server/brokenLinks.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/brokenLinks.js b/node_modules/@docusaurus/core/lib/server/brokenLinks.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/clientModules.d.ts b/node_modules/@docusaurus/core/lib/server/clientModules.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/clientModules.js b/node_modules/@docusaurus/core/lib/server/clientModules.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/codegen/codegen.d.ts b/node_modules/@docusaurus/core/lib/server/codegen/codegen.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/codegen/codegen.js b/node_modules/@docusaurus/core/lib/server/codegen/codegen.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/codegen/codegenRoutes.d.ts b/node_modules/@docusaurus/core/lib/server/codegen/codegenRoutes.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/codegen/codegenRoutes.js b/node_modules/@docusaurus/core/lib/server/codegen/codegenRoutes.js
old mode 100644
new mode 100755
index 74d7218..25cf8b7
--- a/node_modules/@docusaurus/core/lib/server/codegen/codegenRoutes.js
+++ b/node_modules/@docusaurus/core/lib/server/codegen/codegenRoutes.js
@@ -199,7 +199,7 @@ ${Object.entries(registry)
     .sort((a, b) => a[0].localeCompare(b[0]))
     .map(([chunkName, modulePath]) => 
 // modulePath is already escaped by escapePath
-`  "${chunkName}": [() => import(/* webpackChunkName: "${chunkName}" */ "${modulePath}"), "${modulePath}", require.resolveWeak("${modulePath}")],`)
+`  "${chunkName}": [() => import(/* webpackChunkName: "${chunkName}" */ "${modulePath}"), "${modulePath}", require.resolve("${modulePath}")],`)
     .join('\n')}};
 `);
 const genRoutesChunkNames = ({ generatedFilesDir, routesChunkNames, }) => (0, utils_1.generate)(generatedFilesDir, 'routesChunkNames.json', JSON.stringify(routesChunkNames, null, 2));
diff --git a/node_modules/@docusaurus/core/lib/server/config.d.ts b/node_modules/@docusaurus/core/lib/server/config.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/config.js b/node_modules/@docusaurus/core/lib/server/config.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/configValidation.d.ts b/node_modules/@docusaurus/core/lib/server/configValidation.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/configValidation.js b/node_modules/@docusaurus/core/lib/server/configValidation.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/getHostPort.d.ts b/node_modules/@docusaurus/core/lib/server/getHostPort.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/getHostPort.js b/node_modules/@docusaurus/core/lib/server/getHostPort.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/htmlTags.d.ts b/node_modules/@docusaurus/core/lib/server/htmlTags.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/htmlTags.js b/node_modules/@docusaurus/core/lib/server/htmlTags.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/i18n.d.ts b/node_modules/@docusaurus/core/lib/server/i18n.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/i18n.js b/node_modules/@docusaurus/core/lib/server/i18n.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/actions.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/actions.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/actions.js b/node_modules/@docusaurus/core/lib/server/plugins/actions.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/configs.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/configs.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/configs.js b/node_modules/@docusaurus/core/lib/server/plugins/configs.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/init.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/init.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/init.js b/node_modules/@docusaurus/core/lib/server/plugins/init.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/moduleShorthand.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/moduleShorthand.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/moduleShorthand.js b/node_modules/@docusaurus/core/lib/server/plugins/moduleShorthand.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/pluginIds.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/pluginIds.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/pluginIds.js b/node_modules/@docusaurus/core/lib/server/plugins/pluginIds.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/plugins.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/plugins.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/plugins.js b/node_modules/@docusaurus/core/lib/server/plugins/plugins.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/pluginsUtils.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/pluginsUtils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/pluginsUtils.js b/node_modules/@docusaurus/core/lib/server/plugins/pluginsUtils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/presets.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/presets.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/presets.js b/node_modules/@docusaurus/core/lib/server/plugins/presets.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/routeConfig.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/routeConfig.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/routeConfig.js b/node_modules/@docusaurus/core/lib/server/plugins/routeConfig.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/synthetic.d.ts b/node_modules/@docusaurus/core/lib/server/plugins/synthetic.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/plugins/synthetic.js b/node_modules/@docusaurus/core/lib/server/plugins/synthetic.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/routes.d.ts b/node_modules/@docusaurus/core/lib/server/routes.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/routes.js b/node_modules/@docusaurus/core/lib/server/routes.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/site.d.ts b/node_modules/@docusaurus/core/lib/server/site.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/site.js b/node_modules/@docusaurus/core/lib/server/site.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/siteMessages.d.ts b/node_modules/@docusaurus/core/lib/server/siteMessages.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/siteMessages.js b/node_modules/@docusaurus/core/lib/server/siteMessages.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/siteMetadata.d.ts b/node_modules/@docusaurus/core/lib/server/siteMetadata.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/siteMetadata.js b/node_modules/@docusaurus/core/lib/server/siteMetadata.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/storage.d.ts b/node_modules/@docusaurus/core/lib/server/storage.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/storage.js b/node_modules/@docusaurus/core/lib/server/storage.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/translations/translations.d.ts b/node_modules/@docusaurus/core/lib/server/translations/translations.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/translations/translations.js b/node_modules/@docusaurus/core/lib/server/translations/translations.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/translations/translationsExtractor.d.ts b/node_modules/@docusaurus/core/lib/server/translations/translationsExtractor.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/server/translations/translationsExtractor.js b/node_modules/@docusaurus/core/lib/server/translations/translationsExtractor.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssg.d.ts b/node_modules/@docusaurus/core/lib/ssg/ssg.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssg.js b/node_modules/@docusaurus/core/lib/ssg/ssg.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgExecutor.d.ts b/node_modules/@docusaurus/core/lib/ssg/ssgExecutor.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgExecutor.js b/node_modules/@docusaurus/core/lib/ssg/ssgExecutor.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgNodeRequire.d.ts b/node_modules/@docusaurus/core/lib/ssg/ssgNodeRequire.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgNodeRequire.js b/node_modules/@docusaurus/core/lib/ssg/ssgNodeRequire.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgParams.d.ts b/node_modules/@docusaurus/core/lib/ssg/ssgParams.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgParams.js b/node_modules/@docusaurus/core/lib/ssg/ssgParams.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.d.ts b/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.html.d.ts b/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.html.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.html.js b/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.html.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.js b/node_modules/@docusaurus/core/lib/ssg/ssgTemplate.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgUtils.d.ts b/node_modules/@docusaurus/core/lib/ssg/ssgUtils.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/ssg/ssgUtils.js b/node_modules/@docusaurus/core/lib/ssg/ssgUtils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/aliases/index.d.ts b/node_modules/@docusaurus/core/lib/webpack/aliases/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/aliases/index.js b/node_modules/@docusaurus/core/lib/webpack/aliases/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/base.d.ts b/node_modules/@docusaurus/core/lib/webpack/base.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/base.js b/node_modules/@docusaurus/core/lib/webpack/base.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/client.d.ts b/node_modules/@docusaurus/core/lib/webpack/client.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/client.js b/node_modules/@docusaurus/core/lib/webpack/client.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/configure.d.ts b/node_modules/@docusaurus/core/lib/webpack/configure.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/configure.js b/node_modules/@docusaurus/core/lib/webpack/configure.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/ChunkAssetPlugin.d.ts b/node_modules/@docusaurus/core/lib/webpack/plugins/ChunkAssetPlugin.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/ChunkAssetPlugin.js b/node_modules/@docusaurus/core/lib/webpack/plugins/ChunkAssetPlugin.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/CleanWebpackPlugin.d.ts b/node_modules/@docusaurus/core/lib/webpack/plugins/CleanWebpackPlugin.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/CleanWebpackPlugin.js b/node_modules/@docusaurus/core/lib/webpack/plugins/CleanWebpackPlugin.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/ForceTerminatePlugin.d.ts b/node_modules/@docusaurus/core/lib/webpack/plugins/ForceTerminatePlugin.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/ForceTerminatePlugin.js b/node_modules/@docusaurus/core/lib/webpack/plugins/ForceTerminatePlugin.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/StaticDirectoriesCopyPlugin.d.ts b/node_modules/@docusaurus/core/lib/webpack/plugins/StaticDirectoriesCopyPlugin.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/plugins/StaticDirectoriesCopyPlugin.js b/node_modules/@docusaurus/core/lib/webpack/plugins/StaticDirectoriesCopyPlugin.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/server.d.ts b/node_modules/@docusaurus/core/lib/webpack/server.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/server.js b/node_modules/@docusaurus/core/lib/webpack/server.js
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/templates/dev.html.template.ejs b/node_modules/@docusaurus/core/lib/webpack/templates/dev.html.template.ejs
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/utils/getHttpsConfig.d.ts b/node_modules/@docusaurus/core/lib/webpack/utils/getHttpsConfig.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@docusaurus/core/lib/webpack/utils/getHttpsConfig.js b/node_modules/@docusaurus/core/lib/webpack/utils/getHttpsConfig.js
old mode 100644
new mode 100755
